package.json

```diff
@@ -37,51 +37,54 @@
-    "@angular/forms": ">=6.1.0",
-    "@angular/http": ">=6.1.0",
-    "@angular/platform-browser": ">=6.1.0",
-    "@angular/platform-browser-dynamic": ">=6.1.0",
-    "@angular/router": ">=6.1.0",
-    "@ng-bootstrap/ng-bootstrap": "^2.0.0",
-    "bootstrap-icons": "^1.1.0",
-    "core-js": "^2.5.4",
+    "@angular/animations": "^17.1.1",
+    "@angular/common": "^17.1.1",
+    "@angular/compiler": "^17.1.1",
+    "@angular/core": "^17.1.1",
+    "@angular/forms": "^17.1.1",
+    "@angular/platform-browser": "^17.1.1",
+    "@angular/platform-browser-dynamic": "^17.1.1",
+    "@angular/router": "^17.1.1",
+    "@ng-bootstrap/ng-bootstrap": "^16.0.0",
+    "bootstrap-icons": "^1.11.3",
     "detect-passive-events": "^2.0.3",
-    "rxjs": "^6.5.2",
-    "zone.js": "^0.8.26"
+    "rxjs": "^7.8.1",
+    "tslib": "^2.6.2",
+    "zone.js": "~0.14.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/cli": "~6.2.9",
-    "@angular/compiler-cli": "^6.1.0",
-    "@angular/language-service": "^6.1.0",
-    "@types/jasmine": "~2.8.8",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "14.6.2",
-    "codelyzer": "^4.3.0",
+    "@angular-devkit/build-angular": "^17.1.1",
+    "@angular-eslint/builder": "17.2.1",
+    "@angular-eslint/eslint-plugin": "17.2.1",
+    "@angular-eslint/eslint-plugin-template": "17.2.1",
+    "@angular-eslint/schematics": "17.2.1",
+    "@angular-eslint/template-parser": "17.2.1",
+    "@angular/cli": "^17.1.1",
+    "@angular/compiler-cli": "^17.1.1",
+    "@types/jasmine": "~5.1.4",
+    "@types/node": "^20.11.13",
+    "@typescript-eslint/eslint-plugin": "6.19.0",
+    "@typescript-eslint/parser": "6.19.0",
+    "codelyzer": "^6.0.2",
     "escape-html": "^1.0.3",
-    "jasmine-core": "~2.99.1",
-    "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~3.0.0",
-    "karma-chrome-launcher": "~2.2.0",
-    "karma-coverage-istanbul-reporter": "~2.0.1",
-    "karma-jasmine": "~1.1.2",
-    "karma-jasmine-html-reporter": "^0.2.2",
-    "mkdirp": "^0.5.1",
-    "ng-packagr": "^3.0.0",
-    "prismjs": "^1.14.0",
-    "protractor": "~5.4.0",
+    "eslint": "^8.56.0",
+    "jasmine-core": "~5.1.1",
+    "jasmine-spec-reporter": "~7.0.0",
+    "karma": "~6.4.2",
+    "karma-chrome-launcher": "~3.2.0",
+    "karma-coverage-istanbul-reporter": "~3.0.3",
+    "karma-jasmine": "~5.1.0",
+    "karma-jasmine-html-reporter": "^2.1.0",
+    "mkdirp": "^3.0.1",
+    "ng-packagr": "^17.1.2",
+    "prismjs": "^1.29.0",
+    "protractor": "~7.0.0",
     "protractor-console-plugin": "^0.1.1",
-    "rimraf": "^2.6.2",
-    "scss-bundle": "^2.4.0",
-    "ts-node": "~7.0.0",
-    "tsickle": "~0.32.1",
-    "tslint": "~5.9.1",
-    "typedoc": "^0.15.8",
-    "typescript": "^2.7.0",
-    "webdriver-manager": "^12.1.7"
+    "rimraf": "^5.0.5",
+    "scss-bundle": "^3.1.2",
+    "ts-node": "~10.9.2",
+    "typedoc": "~0.25.7",
+    "typescript": "~5.2.2",
+    "webdriver-manager": "^12.1.9"
```

Comment:
`webdriver-manager` pulls in deprecated `requests` and `uuid`. But it is probably fine, as it is only for e2e tests.

Reply:
Never mind, I have a solution for it on follow-up branch `ng17`.